modules/gitops-release-notes-1-14-0.adoc

```diff
@@ -105,4 +130,14 @@ data:
 <1> Specifies the name of the ApplicationSet GitLab SCM Provider.
 <2> Specifies the namespace where the ApplicationSet GitLab SCM Provider is defined.
 +
-For more information, see link:https://argocd-operator.readthedocs.io/en/latest/reference/argocd/#add-self-signed-tls-certificate-for-gitlab-scm-provider-to-applicationsets-controller[Add Self-signed TLS Certificate for Gitlab SCM Provider to ApplicationSet Controller].
\ No newline at end of file
+For more information, see link:https://argocd-operator.readthedocs.io/en/latest/reference/argocd/#add-self-signed-tls-certificate-for-gitlab-scm-provider-to-applicationsets-controller[Add Self-signed TLS Certificate for Gitlab SCM Provider to ApplicationSet Controller].
+
+[id="breaking-change-1-14-0_{context}"]
+== Breaking change
+
+[id="cluster-secrets-with-empty-values_{context}"]
+=== Cluster secrets with empty secret values
+
+* When you upgrade to {gitops-title} v1.14, cluster secrets with an empty project value will no longer be classified as global secrets applicable to all `Applications` and `ApplicationSets`. In previous versions of {gitops-shortname}, any `Application` or `ApplicationSet` could use a cluster secret that matches the URL specified in the `repoUrl` field. Starting from {gitops-title} v1.14, only cluster secrets scoped to an application's project will be considered for matching a repository URL. As a result, if a cluster secret is designated to project A, an application assigned to project B will be unable to access that secret. To allow applications across various projects to use cluster secrets, you must remove the `project` field. link:https://issues.redhat.com/browse/GITOPS-5623[GITOPS-5623]
+
+For more information, see link:https://argo-cd.readthedocs.io/en/latest/operator-manual/upgrading/2.11-2.12/#cluster-secret-scoping-changes[Cluster secret scoping changes].
```

Comment:
- Suggest breaking this up for readability. Use the `+` between lines to make sure everything is tacked on (missing in the "For more information" line as drafted).
- Should avoid using `Applications` and `ApplicationSets` as nouns. I am not familiar with this product area, but would something like "`Application` and `ApplicationSet` resources" work?
- Also recommend moving the bug link to the end to match usual format.

```suggestion
* When you upgrade to {gitops-title} v1.14, cluster secrets with an empty project value are no longer classified as global secrets applicable to all `Applications` and `ApplicationSets`. 
+
In previous versions of {gitops-shortname}, any `Application` or `ApplicationSet` could use a cluster secret that matches the URL specified in the `repoUrl` field. Starting from {gitops-title} v1.14, only cluster secrets scoped to an application's project will be considered for matching a repository URL. 
+
As a result, if a cluster secret is designated to project A, an application assigned to project B will be unable to access that secret. To allow applications across various projects to use cluster secrets, you must remove the `project` field. 
+
For more information, see link:https://argo-cd.readthedocs.io/en/latest/operator-manual/upgrading/2.11-2.12/#cluster-secret-scoping-changes[Cluster secret scoping changes]. link:https://issues.redhat.com/browse/GITOPS-5623[GITOPS-5623]
```

Reply:
Fixed.